packages/dashboard/src/components/contextMenu/menu.tsx

```diff
@@ -49,16 +55,14 @@ const Menu: React.FC<MenuProps> = ({ position, clickOutside, children }) => {
         style={{
           ...styles.popper,
           zIndex: position.z,
+          border: `${spaceScaledXxxs} solid ${colorBorderControlDefault}`,
+          minWidth: '192px', // 128+64
```

Comment:
Is this from the previous implementation? What's the relevance of 128 + 64?

Reply:
instead of 200px, just having a width which is a multiple of 8...

Reply:
can you add a note for that incase we refactor in the future? Then we know it wasn't specifically required to make something else work and can safely be increased / decreased?

Reply:
updated